app/lib/three_scale/analytics/user_tracking.rb

```diff
@@ -6,10 +6,19 @@
 
 module ThreeScale
   module Analytics
+
+    class UserTrackingError < StandardError
+      def initialize(status, error)
+        msg = "User tracking report failed with status: #{status}"
+        msg << ", message: #{error}" if error
```

Comment:
is `error` an `Exception`? Or is it just a string? If it is an `Exception` in this way we would lose the actual backtrace that will help us debug.

Reply:
This is not an exception. In theory it should be a string, but testing actually showed that it is `nil` (probably because we use some old version).
That's why I added the line `:13`.